packages/element/README.md

```diff
@@ -265,9 +265,7 @@ This is the same concept as the React Native implementation.
 
 _Related_
 
--   <https://facebook.github.io/react-native/docs/platform-specific-code#platform-module>
-
-Here is an example of how to use the select method:
+-   <https://facebook.github.io/react-native/docs/platform-specific-code#platform-module> Here is an example of how to use the select method:
```

Comment:
Line breaks get removed.

Reply:
The example intro is bundled as part of the `@see` tag description while is a separate thing. I presume `comment-parser` assigns all the text within tags to the previous tag, hence the error.

One thing we can do to fix this is moving the intro text `Here is an example of how to use the select method:` below the `@example` tag. That way it works fine.